docs/user_guide.md

```diff
@@ -0,0 +1,158 @@
+# User Guide
+
+<div id=body></div>
+

```

Comment:
Can we add a static `<p>` at the top with a brief preamble, e.g., "This page contains the complete list of "tips" available through JetUML's "Tip of the Day" dialog. Click on an entry to expand with details.".